test/parallel/test-timers-setunreftimeout.js

```diff
@@ -0,0 +1,60 @@
+// Flags: --expose-internals
+
+'use strict';
+
+const common = require('../common');
+
+const { strictEqual } = require('assert');
+const { setUnrefTimeout } = require('internal/timers');
+
+{
+  common.expectsError(
+    () => setUnrefTimeout(),
+    {
+      code: 'ERR_INVALID_CALLBACK',
+      type: TypeError,
+      message: 'Callback must be a function'
+    }
+  );
+}
+
+// Test this if correctly passes arguments to the callback
+{
+  const maxArgsNum = 4;
+  for (let i = 0; i < maxArgsNum; i++) {
+    const results = [];
+    const inputArgs = [];
+    // set the input argument params
+    for (let j = 0; j <= i; j++) {
+      inputArgs.push(j);
+    }
+
+    // For checking the arguments.length,
+    // callback function should not be arrow fucntion.
+    const timer = setUnrefTimeout(common.mustCall(
+      function(arg1, arg2, arg3, arg4) {
```

Comment:
Also, I think you could use here
`...args`.
Then later check `args.length`
and instead of pushing one-by-one use 
```
for (const arg of args) results.push(arg);

of 

args.forEach(arg => results.push(args));
```

Though it's also fine as it is now.

Reply:
Thank you so much. I'll fix it